.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: node_js
+node_js: 8
 sudo: false
-node_js:
-- '8'
-before_install:
-- bundle install
+
 install:
-- npm install
-script:
-  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
-  - chmod 755 buildAndDeploy.sh
-  - travis_wait 30 sleep infinity & ./buildAndDeploy.sh
+  - npm install
+  - bundle install
```

Comment:
Might this be a good place to add a `- npm test`?

Reply:
haha nice catch, I didn't even noticed the test command was not even used.

Reply:
`npm test` is the default `script`, i.e. main CI phase for projects with `language: node_js`

Reply:
I would prefer to make it explicit in the CI configuration, people changing this in the future (looking in the mirror: me!) might now know about it and wonder what is going on. Doesn't hurt having it, does it?

Reply:
It hurts my eyes! :scream:

JK, it's definitely more obvious with the `script` phase explicitly defined.